readium/navigator/src/main/java/org/readium/r2/navigator/input/InputListener.kt

```diff
@@ -9,17 +8,17 @@ public interface InputListener {
      * Called when the user tapped the content, but nothing handled the event internally (eg.
      * by following an internal link).
      */
-    public fun onTap(navigator: VisualNavigator, event: TapEvent): Boolean = false
```

Comment:
I simplified `InputListener` to not depend on a `Navigator` instance. This way it could be forwarded to a lower level component, such as a `PdfDocumentFragment`.